src/cpu/mod.rs

```diff
@@ -14,12 +14,12 @@ enum Instruction {
     AddImm(),
     AddHL(),
     Adc(ArithmeticOperand),
-    // TODO: AdcHL
+    AdcHL(),
```

Comment:
Maybe we wanna change the name of these to match Ld.  If we go with LdFromMem maybe AdcFromMem or something.

Reply:
I'd hate to admit it, but when you're right you're right. Go ahead. The variable name must not be an obstacle.

Reply:
Added 3 new commits to address this.  You know me, all or nothin baby.